Lib/ufo2ft/fastUFO.py

```diff
@@ -13,9 +13,26 @@
 # limitations under the License.
 
 
-from fontTools.pens.boundsPen import BoundsPen
+from fontTools.pens.boundsPen import BoundsPen, ControlBoundsPen
 from ufoLib import UFOReader, UFOWriter
 from ufoLib.pointPen import PointToSegmentPen, SegmentToPointPen
+import os
+import weakref
+try:
+    from collections.abc import Mapping  # python >= 3.3
+except ImportError:
+    from collections import Mapping
```

Comment:
I believe `from collections import Mapping` should work on python 2 and 3. At least, on the pythons that I have installed locally (2.7.13, 3.5.4, and 3.6.2).